### DISCUSSION TITLE
Share configs

### DISCUSSION THREAD

**Discussion by debuxxed** [linearmouse/linearmouse#821] [Q&A]
I've got Keychron M3 mouse, and the problem is that the app distinguishes the mouse when it's in different states of mouse
For example:
- Keychron Link (Dongle Mode)
- Keychron M3 (Charging USB Type-C mode).

Any way to share configs for both of them?

**Comment by lujjjh** [comment]
Yes, you can do this by manually editing the configuration:

https://github.com/linearmouse/linearmouse/blob/main/Documentation/Configuration.md#schemes-merging-and-multiple-ifs
